src/yunohost/hook.py

```diff
@@ -447,10 +447,10 @@ def _hook_exec_bash(path, args, chdir, env, user, return_format, loggers):
             raw_content = f.read()
         returncontent = {}
 
-        if return_format == "json":
+        if return_format == "yaml":
```

Comment:
Normally, json is included in yaml, so if a script return a json, it could be read with read_yaml.

However as it's not the same parser, that's true that some small differiencies could exists.

For now i don't even know if some hook returns json ?

Reply:
I made this change cause it's clearly easier to write yaml in bash than json.